packages/airnode-validator/test/fixtures/ois.json

```diff
@@ -52,6 +52,7 @@
         "method": "get",
         "path": "/convert"
       },
+      "cacheResponses": "false",
```

Comment:
Should we make this more configurable? (E.g. how long to cache a value)

Reply:
I disagree that this should be in OIS. OIS is an abstraction that shouldn't know anything about Airnode. This flag should probably go to `triggers.rrp[n].cacheResponses`

Reply:
Under that suggestion wouldn't we also need `cacheResponses` in the gateways `http[n]` and `httpSignedData[n]`? It also seems like it would be easy to mistakenly configure caching inconsistently between triggers then.

Reply:
> Under that suggestion wouldn't we also need cacheResponses in the gateways http[n] and httpSignedData[n]?

Yeah, it needs to go there, because it can't go to OIS and I don't see a better place.

> It also seems like it would be easy to mistakenly configure caching inconsistently between triggers then.

You mean having 2 triggers for the same endpoint and only one of them has caching enabled? Not sure if this is a feature or a bug though. We may open a slack discussion about this, because I see that this whole discussion can be opinionated... One might argue that caching is a generic concept and advocate to put it on OIS.

Reply:
Resolving per the agreement to have it in `triggers.rrp[n]` and not cache gateway responses.